javascript/myportfolio.js

```diff
@@ -203,6 +203,23 @@ contactForm.addEventListener('submit', (event) => {
   }
 });
 
+// .................Preserving data using local storage........
+
+const clientName = document.getElementById('name');
```

Comment:
- [ ] Kindly ensure that your saving to local storage functionality is working before you submit. I can see that you are using const to declare `clientName` in line 298 and 225 and this could be causing some problems.